### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by amirnathani9**
## changelog
- added wishlist

## screenshots
image

## preview link
https://deploy-preview-5--watchesworld.netlify.app/pages/wishlist.html

### CODE REVIEW

**Review by HarshnaNHaswani — APPROVED** [review]
Looks good to me, just format your file a bit

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        <i class="fas fa-search navbar-search-btn"></i>
+      </div>
+      <ul class="navbar-links">
+        <li>
+          <a href="" class="border-radius-2"
+            ><i class="fas fa-user"></i>Logout</a
+          >
+        </li>
+        <li>
+          <a href="" class="border-radius-2"
```
Comment (HarshithaKonduru214) [inline pages/wishlist.html:49]: Format the code a bit so that it is easier to read.

**Review by HarshithaKonduru214 — APPROVED** [review]
Looks good to me.

File: pages/wishlist.html (hunk 1 above)
Comment (amirnathani9) [reply pages/wishlist.html:49]: ok i will look into it

**Pull request merged**